### PR TITLE
[BRD-144] - Loading Page

### PR DESCRIPTION
**Pull request by Evange95**
created loading page and swapped with skeleton

### CODE REVIEW

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        44.368421% {
+            transform: translate(256.202551px, 150.000005px);
+            fill: var(--microlc-primary-color-tint-89);
+        }
+
+        60.157895% {
+            transform: translate(256.202551px, 88.001635px);
+            fill: var(--microlc-primary-color);;
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:29]: use a single ;

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        75.947368% {
+            transform: translate(149.999901px, 90.045537px);
+            fill: var(--microlc-primary-color);;
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:34]: use a single ;

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        91.736842% {
+            transform: translate(150.000001px, 150.000005px);
+            fill: var(--microlc-primary-color);;
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:39]: use a single ;

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+        100% {
+            transform: translate(150.000001px, 150.000005px);
+            fill:var(--microlc-primary-color-tint-89);
+        }
+    }
+
+    #ep3zs54gtu13_to {
+        animation: ep3zs54gtu13_to__to 2100ms linear infinite normal forwards;
+        fill :var(--microlc-primary-color-tint-89)
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:50]: use a single ;

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    @keyframes ep3zs54gtu13_to__to {
+        0% {
+            transform: translate(43.797552px, 150.000004px);
+            fill: var(--microlc-primary-color);;
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:56]: use a single ;

(check the rest of the file)

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.test.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react'
+import {screen} from '@testing-library/react'
+import {LoadingAnimation} from './LoadingAnimation'
+import RenderWithReactIntl from '../../__tests__/utils'
+
+describe('ErrorPageContainer tests', () => {
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.test.tsx:21]: this is not a test for error page container :smiley:

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import {ReactComponent as SvgAnimation} from './assets/LoadingAnimation.svg'
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.tsx:18]: use a space between these 2 imports

File: packages/fe-container/src/containers/launcher/Launcher.tsx [hunk 8]
```diff
@@ -32,7 +33,7 @@ export const Launcher: React.FC<AppState> = ({configuration, isLoading, user}) =
     <>
       {
         isLoading ?
-          <Skeleton.Input active className='launcher_skeleton'/> :
+        <LoadingAnimation /> :
```
Comment (ilteoood) [inline packages/fe-container/src/containers/launcher/Launcher.tsx:36]: please re-align

**Review by ilteoood — CHANGES_REQUESTED** [review]

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+    width          : 100%;
+    align-items    : center;
+    justify-content: center;
+    display        : flex;
+    flex-direction : column;
+
+
+    #ep3zs54gtu12_to {
+        animation: ep3zs54gtu12_to__to 2100ms linear infinite normal forwards;
+        fill :var(--microlc-primary-color-tint-89)
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:13]: here is missing the semi column

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.less [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+            transform: translate(256.202452px, 150.000004px);
+            fill:var(--microlc-primary-color-tint-89);
+        }
+    }
+
+    
+
+    #ep3zs54gtu14_to {
+        animation: ep3zs54gtu14_to__to 2100ms linear infinite normal forwards;
+        fill :var(--microlc-primary-color-tint-89)
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.less:99]: semicolumn missing

File: packages/fe-container/src/components/loading-animation/LoadingAnimation.test.tsx [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react'
+import {screen} from '@testing-library/react'
+import {LoadingAnimation} from './LoadingAnimation'
```
Comment (ilteoood) [inline packages/fe-container/src/components/loading-animation/LoadingAnimation.test.tsx:18]: insert a space between these 2 imports

**Review by ilteoood — CHANGES_REQUESTED** [review]

**Review by epessina — APPROVED** [review]

**Pull request merged**